toptal/picasso
# [FX-NULL] Add Version Packages reminder

### Description

This pull request adds GHA workflow that checks if there are open Version Packages pull request and, if there are any, reports it in channel.

To avoid extra costs, it runs only on workdays during working hours (approximately).

### How to test


- [Temploy](https://picasso.toptal.net/FX-NULL-add-version-packages-pull-request-reminder)
- Please see the https://github.com/toptal/picasso/actions/workflows/notify-about-pending-release-pull-requests.yaml runs

### Screenshots

<img width="552" alt="Screenshot 2024-08-22 at 20 28 09" src="https://github.com/user-attachments/assets/2e790605-360c-4f71-b8e3-30edf3477d1a">

### Development checks

- [N/A] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [N/A] Make sure that additions and changes on the design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core
- [x] ⚠️ **Address all `TODO` before merge**

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>
